scss/message/_message.scss

```diff
@@ -440,6 +440,18 @@
     }
   }
 
+  &.webxdc-info > .bubble {
+    padding: 7px 8px;
+    img {
+      height: 18px;
+      width: 18px;
+      margin-right: 5px;
```

Comment:
at some point, we will probably add rtl-support, then we should switch to `margin-inline` or so. just saying, no need to start with that now.